javascript/operators/await.json

```
@@ -62,6 +62,75 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "top_level": {
+          "__compat": {
+            "description": "Use at module top level",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/await",
+            "spec_url": "https://tc39.es/proposal-top-level-await/#introduction",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "notes": "See <a href='https://crbug.com/9344'>bug 9344</a>"
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "See <a href='https://bugzil.la/1519100'>bug 1519100</a>"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": [
+                {
+                  "version_added": "14.8.0",
+                  "notes": "Not supported in CommonJS modules."
+                },
+                {
+                  "version_added": "14.3.0",
+                  "flags": [
+                    {
+                      "type": "runtime_flag",
+                      "name": "--experimental-top-level-await"
+                    }
+                  ],
+                  "notes": "Not supported in CommonJS modules."
+                }
+              ],
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false,
+                "notes": "See <a href='https://webkit.org/b/202484'>bug 202484</a>"
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```

It looks like there's only one implementation right now. And I guess stage 3 proposals could conceivably change. Can we flip this one?

```suggestion
              "experimental": true,
```